DesignSafe-CI/portal
# Task/des 2339 passing new data to datacite

## Overview: ##
Update Metadata that is being passed to Datacite
(1) Send Project Title as titleType 'Subtitle'

      "titles": [
    {
      "title": "mission title"
    },
    {
      "title": "Vanessa Field Research",
      "titleType": "Subtitle"
    }
    ],


(2) Send Project description as descriptionType 'Abstract'

     "descriptions": [
    {
      "lang": "en-Us",
      "description": "mission description mission description mission description",
      "descriptionType": "Abstract"
    },
    {
      "lang": "en-Us",
      "description": "description asdf",
      "descriptionType": "Abstract"
    }
    ],

(3) Send Award Title and Number from project level

    "fundingReferences": [
       {
         "awardTitle": "test award",
         "awardNumber": "13245"
       }
    ],

## PR Status: ##


## Related Jira tickets: ##

* [DES-2339](https://jira.tacc.utexas.edu/browse/DES-2339)

## Summary of Changes: ##
For all project types:
1. Pass project info to to_datacite_json.
2. Include title, description, and award info to datacite metadata.
3. Pass facility info using condition that takes NO facility into account.

## Testing Steps: ##
1. Create a new project or use an old project: (check that all 3 options work)
 - Re-publish 
 - Amend
 - Version
2. Go to DataCite Test (https://doi.test.datacite.org/repositories/tdl.tacc) check for updated data. (Password in Stache)

## UI Photos:

## Notes: ##
This should work with or without a facility. 
There is a following up ticket that requires modal changes, then updated data to Datacite: [DES-2603](https://jira.tacc.utexas.edu/browse/DES-2603)
